vuejs/vue-cli
# feat: allow choosing vue version on creation (and in presets)

This PR:
- adds a `vueVersion` field in the preset schema (accepts a string, defaults to `'2'`, and can be `'3'`).
- adds a new "Default (Vue 3 Preview)" preset
- allow choosing Vue version on `vue create` manual mode

Progress:
- [x] new preset field
- [x] `vue create` prompt
- [x] `@vue/cli-service`
- [x] ESLint plugin
- [x] Router plugin
- [x] Vuex plugin
- [x] TypeScript
- [x] Babel JSX support (merged #5665)
- [x] Jest plugin
- [x] Mocha plugin
- [x] Cypress plugin
- [x] NightWatch plugin
- [x] Tests for these templates
- [x] Prompts in UI
- [x] Test for prompts
- [x] <http://link.vuejs.org/feature-flags>

Manually tested (should write unit tests later):
- [x] `vue invoke` in a Vue 3 project
- [x] JSX & TSX

Maybe later:
- [ ] Wrap object exports with `defineComponent` calls in `.vue` files


**What kind of change does this PR introduce?** (check at least one)

- [ ] Underlying tools


**Does this PR introduce a breaking change?** (check one)


**Other information:**
